.gitignore

```diff
@@ -67,3 +67,6 @@ integration-test/transform-tests/output
 
 # generated Sphinx/ReadTheDocs files 
 /docs/build/
+
+# generated as part of npm release process
```

Comment:
maybe add a note here that the file is necessary for tests too, which is not usually the case for `dist/` files